src/components/ShareTrajectoryButton/index.tsx

```diff
@@ -41,19 +45,25 @@ const ShareTrajectoryButton = ({
             ) : null}
             <Tooltip
                 title={
-                    isDisabled
+                    !simulariumFile.name
                         ? "Load a model to perform this action"
                         : "Share trajectory"
                 }
                 placement="bottomLeft"
                 color={TOOLTIP_COLOR}
                 align={{ offset: tooltipOffset }}
+                mouseEnterDelay={0.5}
+                trigger={isSharing ? [] : ["hover"]}
+                onOpenChange={(visible) => {
+                    setTooltipVisible(visible);
+                }}
```

Comment:
If you move the cursor on the button quickly and click right away `handleShare` runs during the `mouseOver` delay and then `onOpenChange` opens the tooltip while the modal is open. Suggestion below seems to fix it, probably other ways as well. Sorry if you get two notifications, put this comment/suggestion on the wrong line a moment ago.
```suggestion
                onOpenChange={(visible) => {
                     if (isSharing) return;
                    setTooltipVisible(visible);
                }}
```

Reply:
I'll do a change like this, but I actually haven't been able to get this to happen

Reply:
Weird, can show a video if you like, maybe its a browser thing if you're on Safari? I'm on Chrome...